### DISCUSSION TITLE
Create event and reminders in the menubar

### DISCUSSION THREAD

**Discussion by BERT-ENTPE** [pakerwreah/Calendr#207] [Ideas]
Hi 👋
I think it would be kinda cool to be able to add tasks and events from the app. 
I don't know if it requires too much time for you, I hope not.
Have a lovely day and thanks for your app, it's really dope.

**Comment by pakerwreah** [comment]
Hi 👋🏻 
Yes, it requires a considerable effort because Apple does not provide a native UI to do that on macOS as they do for iOS.
Since creating new events is not really the focus of this app, it's not worth spending much time creating a UI from scratch with all the available features from the native calendar.

**Reply by BERT-ENTPE** [reply]
oh, okay ^^ I understand :) thanks for your message and have a great day.

**Comment by l16i** [comment]
I'd like to add my two cents if this feature is ever planned and put it here as reference if it's needed one day.

One app has an event creation popover view that works well, which might be helpful in the future.


The code is:

the event creation view
https://github.com/sfsam/Itsycal/blob/master/Itsycal/EventViewController.m

the calling of the popover view
https://github.com/sfsam/Itsycal/blob/55650e2592b2bccdcd2a3bc5edeedd6939644633/Itsycal/ViewController.m#L216-L279

**Reply by pakerwreah** [reply]
The problem is not the code, it's the time and effort to maintain it.
It will never provide all the features that the native app does, and people will keep asking to add more and more stuff to make it look identical.
You can literally create an event by just double-clicking a date in Calendr and using the native app, there's no need to reinvent the wheel.
The day Apple decides to expose the native View Controller as they do for iOS, I will be for sure one of the first devs to use it.

**Reply by l16i** [reply]
I understand, and it makes sense.
I shared this in case one of the developers (not necessarily you) wanted to implement it, but you're right that maintenance must be taken into account.
Thanks for the detailed response!
